### PR TITLE
add CoC and co to README

### PR DESCRIPTION
**Pull request by sebgoa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,25 @@ command:
 $ ./til --help
 ```
 
+## Contributions and support
+
+We would love to hear your feedback on these sources. Please don't hesitate to submit bug reports and suggestions by
```
Comment (antoineco) [inline README.md:43]: The wording isn't relevant for this repo, it should be adapted.

Comment (sebgoa) [inline README.md:43]:
```suggestion
We would love to hear your feedback. Please don't hesitate to submit bug reports and suggestions by
```

Comment (sebgoa) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -38,6 +38,25 @@ command:
...
+[filing issues][gh-issue], or contribute by [submitting pull-requests][gh-pr].
+
+## Commercial Support
+
+TriggerMesh Inc. supports TIL commercialy. Email us at <info@triggermesh.com> to get more details.
```
Comment (sebgoa) [inline README.md:48]:
```suggestion
TriggerMesh Inc. supports TIL commercially. Email us at <info@triggermesh.com> to get more details.
```

**Review by antoineco — APPROVED** [review]

**Pull request merged**